### PR TITLE
SolutionSet_2_Naman

### PR DESCRIPTION
**Pull request by IronicalCoder**

### CODE REVIEW

**Comment by paramkpr** [comment]
Remove the separate Q1 and Q2 files and then I'll merge

**Comment by IronicalCoder** [comment]
Ready to Merge

**Comment by paramkpr** [comment]
Nice, use modules next time, I'm merging now.

File: QuestionSets/PythonLanguage/ClassesAndObjects/Naman_SolutionSet2/Class_And_Objects_SolutionSet2Q1+Q2.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        self.health -= 5
+
+    def heal(self):
+        self.health += 5
+
+    def booster(self):
+        self.attack_power += 5
+
+    def attack(self,Player):
+        Player.take_damage
```
Comment (ViRu-ThE-ViRuS) [inline QuestionSets/PythonLanguage/ClassesAndObjects/Naman_SolutionSet2/Class_And_Objects_SolutionSet2Q1+Q2.py]: @IronicalCoder @paramkpr ye kya bhai??

Comment (paramkpr) [reply QuestionSets/PythonLanguage/ClassesAndObjects/Naman_SolutionSet2/Class_And_Objects_SolutionSet2Q1+Q2.py]: @ViRu-ThE-ViRuS Sorry. He told me that it was working... Not listening to @IronicalCoder next time. My bad.

**Pull request merged**